mongo/datadog_checks/mongo/collectors/base.py

```diff
@@ -25,11 +25,15 @@ def __init__(self, check, tags):
         self.base_tags = tags
         self.metrics_to_collect = self.check.metrics_to_collect
 
-    def collect(self, client):
+    def collect(self, api):
```

Comment:
Renames to `api` to clear up that this variable is an instance of the new class `MongoApi`. Because MongoApi extends MongoClient, there are interchangeable in that case.